README.md

```diff
@@ -184,4 +184,64 @@ In this directory (you should see the `test` and `traffic_comparator` directorie
 
 ```
 pytest
-```
\ No newline at end of file
+```
+
+## Exploring Results in Jupyter Notebook/IPython
+
+Currently, comparison results can be reported in very high level reports (average latencies, percent of matching status codes), or in very low level detail (all line-by-line diffs output to a file). Neither of these formats are conducive to data exploration or visualization.
+
+A new option has been added: comparison results can be dumped to a sqlite database, and then read in using some provided functions in a Jupyter Notebook or IPython terminal, and explored and visualized there.
+
+The following is a walk through of the sample `Results Repository.ipynb` file.
+
+### Install Data Exploration-specific Dependencies
+
+```
+pip install --editable ".[data]"
+```
+
+### Dump Comparison Results to SQLite
+
+A command has been added that handles accepting comparison results and dumping them to sqlite in the appropriate format.
+
+If you're doing your comparison at the same time, it can be run as in the following example, with any of the possible settings from above. It supports running in streaming mode and will flush each write to the sqlite db.
+
+```
+cat input_triples.log | trafficcomparator stream | trafficcomparator dump-to-sqlite
+```
+
+If you have already generated comparisons (e.g. run the `stream` command) and saved the output, you can pipe that directly into the `dump-to-sqlite` command.
+
+```
+cat comparison_results.log | trafficcomparator dump-to-sqlite
+```
+
+### Start the Jupyter server and load results
```

Comment:
Nit - Since this is a detailed walkthrough of a specific notebook, consider moving this to a separate README, and keeping this section short:

It is possible to use either Jupyter notebooks (web-based UI to a local server that runs the python kernel) or in an IPython terminal (command line terminal with kernel running directly) to visualize comparison data. For a detailed walkthrough of the sample notebook, see `<markdown-file>`. The Jupyster server can be started with `jupyter notebook`